docs/tutorials/mongoid-documents.txt

```diff
@@ -176,6 +177,38 @@ Types that are not supported as dynamic attributes since they cannot be cast are
 - ``DateTime``
 - ``Range``
 
+The StringifiedSymbol Type
+--------------------------
+
+When using the built-in Symbol field type, Mongoid defaults to serializing the field into the BSON::Symbol
+type. For more information on the BSON::Symbol type, see
+`here <https://docs.mongodb.com/mongoid/master/tutorials/mongoid-documents/#bson-symbol-type>`_.
+BSON::Symbol is deprecated, so the StringifiedSymbol type allows users to write code
+that makes use of the built-in Symbol type while ensuring interoperability with other drivers.
+The StringifiedSymbol type stores all data on the database as a String, while returning values to the
+user as a Symbol.
+
+An example usage is shown below:
+
+.. code-block:: ruby
+
+  class Post
+    include Mongoid::Document
+
+    field :status, type: StringifiedSymbol
+  end
+
+  post = Post.new(status: "hello")
+  post.status # => :hello
+
+All values will be stringified upon being sent to the database, and all values will be converted
+to symbols when returned to the user. This includes values that cannot be converted directly to
+symbols, such as integers and arrays.
+
+This type will lazily migrate any BSON::Symbols already stored in the database to BSON::Strings. If
+the user changes their model to use StringifiedSymbol instead of Symbol, the documents will be saved
```

Comment:
I would try to use indirect language as much as possible, so instead of saying "if the user does xyz," say "If the ``StringifiedSymbol`` type is applied to a field that contains ``BSON::Symbol`` data..."

Reply:
:+1: